src/Widgets/Widgets.md

```diff
@@ -0,0 +1,102 @@
+### Camera Controller
```

Comment:
This code seems to be imported from [UD-Viz-Template](https://github.com/VCityTeam/UD-Viz-Template/blob/master/Readme.md#camera-controller) (because eventually it belongs in UD-Viz).
Provide a PR in UD-Viz-Template that cleans up the source from the content that was moved here.

Reply:
Already fixed by [this PR](https://github.com/VCityTeam/UD-Viz-Template/pull/34)